hypervisor/common/sbuf.c

```diff
@@ -104,3 +113,29 @@ int32_t sbuf_setup_common(struct acrn_vm *vm, uint16_t cpu_id, uint32_t sbuf_id,
 
 	return ret;
 }
+
+/* try put a batch of elememts from data to sbuf
+ * data_size should be equel to n*elem_size, data not enough to fill the elem_size will be ignored.
```

Comment:
It seems that constraint does not hold for the callsites in profiling.c. Shall we properly handle the case where `data_size` is not a multiple of `elem_size`?